Podcast/SeriesType.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  SeriesType.swift
+//  Podcast
+//
+//  Created by Natasha Armbrust on 9/14/17.
+//  Copyright © 2017 Cornell App Development. All rights reserved.
+//
+
+import UIKit
+
+enum SeriesType {
```

Comment:
Do we really need a whole enum for this?

Reply:
No we don't actually and we shouldn't because now one is a subclass of the other so we can just use casting. I'll remove this.